### PR TITLE
Fix : Reduce codacy issues

### PR DESCRIPTION
**Pull request by amitiwary999**
Fixes issue #682

### CODE REVIEW

**Comment by amitiwary999** [comment]
previous pull request was creating  merge conflict.

**Comment by amitiwary999** [comment]
@chiragw15 @chashmeetsingh @mayank408 review.

File: app/src/main/java/org/fossasia/susi/ai/activities/LoginActivity.java [hunk 1]
```diff
@@ -167,70 +151,44 @@ public void onResponse(Call<LoginResponse> call, Response<LoginResponse> respons
...
-                        AlertDialog.Builder builder = new AlertDialog.Builder(LoginActivity.this);
-                        builder.setTitle(response.code() + getApplicationContext().getString(R.string.error));
-                        builder.setMessage(response.message())
-                                .setCancelable(false)
-                                .setPositiveButton(getApplicationContext().getString(R.string.ok), null);
-                        AlertDialog alert = builder.create();
-                        alert.show();
-                        Button pbutton = alert.getButton(DialogInterface.BUTTON_POSITIVE);
-                        pbutton.setTextColor(Color.BLUE);
+                    alertNotSuccess(response.code()+getResources().getString(R.string.error), response.message());
```
Comment (mayank408) [inline app/src/main/java/org/fossasia/susi/ai/activities/LoginActivity.java]: @amitiwary999 As discussed earlier can't we have a single alert for all. Could you please tell me how are you accounting for the callacks.

Comment (amitiwary999) [inline app/src/main/java/org/fossasia/susi/ai/activities/LoginActivity.java]: @mayank408 everything is discussed with @chiragw15 [here](https://github.com/fossasia/susi_android/pull/684). I changed code accordingly.

**Pull request merged**